root/components/common-macros.tt

```diff
@@ -6,6 +6,7 @@
 [%~ MACRO replace(text, search, replace) BLOCK; text | replace(search, replace); END -%]
 [%~ MACRO url_escape(url) BLOCK; url | url; END -%]
 [%~ MACRO html_escape(url) BLOCK; url | html; END -%]
+[%~ MACRO html_unescape(text) BLOCK; text.replace('&quot;', '"').replace('&lt;', '<').replace('&gt;', '>').replace('&#39;', "'").replace('&amp;', '&'); END -%]
```

Comment:
Would it work to just put `html_unescape => &HTML::Entities::decode_entities` in the stash?

I've wondered why we don't do the same for `html_escape`, in order to use our own `encode_entities`.